src/test_typing_extensions.py

```diff
@@ -5377,6 +5390,21 @@ def test_valid_uses(self):
             self.assertEqual(C3.__origin__, C4.__origin__)
             self.assertNotEqual(C3, C4)
 
+        # Callable with Ellipsis cannot be constructed in 3.8 and below 3.9.2
+        if sys.version_info[:2] <= (3, 9, 2):
+            return
+
```

Comment:
There are obviously lots of situations in this file where we already disobey this rule (which arguably we should fix, IMO) -- but in general, it's good to put assertions that only pass on certain versions in separate test methods which are skipped on certain Python versions. If you do so, the test explicitly shows up as being "skipped" in the summary report when you run the tests, rather than the test silently behaving differently on different Python versions.

```suggestion
    @skipUnless(sys.version_info >= (3, 9, 3))
    def test_valid_uses_py39_plus(self):
        # Callable with Ellipsis cannot be constructed in 3.8 and below 3.9.2

```